generators/client/templates/gulpfile.js

```diff
@@ -125,44 +124,40 @@ gulp.task('styles', [<% if(useSass) { %>'sass'<% } %>], function () {
         .pipe(browserSync.reload({stream: true}));
 });
 
-gulp.task('inject', function () {
+gulp.task('inject', ['inject:app','inject:test', 'inject:vendor']);
```

Comment:
@sendilkumarn can you change this as well please as all these shouldn't be run together and in some tasks these needs to be run in certain sequence along with other tasks